app/components/jobseekers/school_group_overview_component.html.haml

```diff
@@ -13,13 +13,19 @@
           %td.govuk-table__cell{ class: "govuk-!-font-weight-bold" }
             = t('school_groups.website')
           %td.govuk-table__cell
-            = link_to(t('jobs.website_link_text', organisation_name: @vacancy.parent_organisation.name), @vacancy.parent_organisation.website.presence || @vacancy.parent_organisation.url, class: 'govuk-link wordwrap', target: '_blank')
+            = link_to(t('schools.website_link_text', organisation_name: @vacancy.parent_organisation.name), @vacancy.parent_organisation.website.presence || @vacancy.parent_organisation.url, class: 'govuk-link wordwrap', target: '_blank')
```

Comment:
Can we save space (and the number of loads?) in the component template, by setting `@trust = vacancy.parent_organisation` in the component initialize method?

Reply:
Or even use an attribute reader if you like, as mentioned in TWD dev meetup!